sites/all/modules/custom/debugacademy/debugacademy.admin.inc

```diff
@@ -0,0 +1,294 @@
+    '#type' => 'fieldset',
+    '#title' => t('Fourth tab content'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );  
+  $form['tabbed_section']['debugacademy_tab_four_pane']['debugacademy_tab_four_pane_title'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Fourth tab title'),
+    '#description' => t('Title of the fourth tab in the four-tab section'),
+    '#default_value' => variable_get('debugacademy_tab_four_pane_title', 'What are the payment options?'),
+    '#required' => TRUE,
+  );
+  $form['tabbed_section']['debugacademy_tab_four_pane']['debugacademy_tab_four_pane_body'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Fourth tab body'),
+    '#description' => t('Body of fourth tab in the four-tab section'),
+    '#default_value' => variable_get('debugacademy_tab_four_pane_body', 'Coming soon.'),
+    '#required' => TRUE,
+  );
+  
+  /*Banners*/
+  $form['banners'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Banners with attention grabbing headlines'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );  
+  $form['banners']['debugacademy_banner_one_headline'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Banner one headline'),
+    '#description' => t('Headline for the first banner.'),
+    '#default_value' => variable_get('debugacademy_banner_one_headline', 'JOIN THE MOVEMENT'),
+    '#required' => TRUE,
+  );
+  $form['banners']['debugacademy_banner_two_headline'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Banner two headline'),
+    '#description' => t('Headline for the second banner.'),
+    '#default_value' => variable_get('debugacademy_banner_two_headline', 'PREVIOUS GRADUATE STATISTICS'),
+    '#required' => TRUE,
+  );
+  
+  /*How Debug Academy is Different*/
+  $form['different'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('How Debug Academy is Different'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );  
+  $form['different']['debugacademy_different_intro'] = array(
+    '#type' => 'textarea',
+    '#title' => t('How Debug Academy is Different'),
+    '#description' => t('Small paragraph on what separates Debug Academy from similar programs.'),
+    '#default_value' => variable_get('debugacademy_different_intro', 'Our personalized approach to teaching allows us to cater to individual student strengths and interests. Students are part of a real development team working on real projects. They leave the class with a working portfolio and applied experience on a software project lead by a senior developer.'),
+    '#required' => TRUE,
+  );
+  $form['different']['debugacademy_different_bullets'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Bulleted list of how Debug Academy is different'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['different']['debugacademy_different_bullets']['debugacademy_different_bullet_one'] = array(
```

Comment:
@malikkotob 
I think it's better to have one long text field than to have an input for each bullet. An input for each bullet is pretty restrictive:
- Have to have a bulleted list (can't change to a paragraph)
- More importantly: we have to have exactly 3 / however many bullets in each section

I think this is a change worth making after the other things you're working on